src/engine/test/source/base/stringUtils_test.cpp

```diff
@@ -75,3 +75,62 @@ TEST(splitMulti, ThreeDelimiters) {
     std::vector<std::string> result = utils::string::splitMulti(input, utils::string::Delimeter('-', true), utils::string::Delimeter(',', false), utils::string::Delimeter(' ', false));
     ASSERT_EQ(result, expected);
 }
+
+TEST(join, defaut_separator)
+{
+    std::string expected = "test";
```

Comment:
Please, add the `const` modifier to every variable that won't change its value. If you have values that are repeated many times you could define a `constexpr` or `const string`, or similar, to avoid redundancy of code.

Reply:
ok